### PR TITLE
Updated to fix imports and time.clock issues, now works with python 3.8

### PR DESCRIPTION
**Pull request by pjaol**
Updated requirments.txt to fix versions
Updated to work with python 3.8
Updated imports to work with utils

### CODE REVIEW

**Comment by pjaol** [comment]
I'm taking a look to see what pulled those in, this is just the output of the pip freeze command. 
WRT versions, it helps with open software as it ensures people are using known working versions and you're not having to figure out what they've installed, are they ahead / behind what you've got etc.. 
I'll see if I can map out the dependencies and wipe my local instance and see what happens
Also did I mention this is really cool :-)

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,11 @@
-SimpleITK
-vtk
-numpy
+astroid==2.4.1
```
Comment (dave3d) [inline requirements.txt:1]: My conda env doesn't have astroid, isort, lazy-object-proxy, mccabe, pylint, toml, or wrapt.

Comment (dave3d) [reply requirements.txt:1]: I'm going to merge the pull request and then edit the requirements.txt file afterwards.

Comment (pjaol) [reply requirements.txt:1]: Hi Dave
Tracked down those dependencies, they came from pylint 

```
pylint==2.5.2
  - astroid [required: >=2.4.0,<=2.5, installed: 2.4.1]
    - lazy-object-proxy [required: ==1.4.*, installed: 1.4.3]
    - six [required: ~=1.12, installed: 1.15.0]
    - wrapt [required: ~=1.11, installed: 1.12.1]
  - isort [required: >=4.2.5,<5, installed: 4.3.21]
  - mccabe [required: >=0.6,<0.7, installed: 0.6.1]
  - toml [required: >=0.7.1, installed: 0.10.1]
setuptools==46.4.0
SimpleITK==1.2.4
vtk==9.0.0
wheel==0.34.2
```

You should be good to go without them. 
Thanks
Patrick

**Pull request merged**